sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_04_select/answers/_09_view_ts.answer

```diff
@@ -28,7 +28,7 @@ col2     TIMESTAMPLTZ     YES          null
 
 ===================================================
```

Comment:
Shouldn't it be changed to the same content below?
```
sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_04_select/answers/_09_view_ts.answer_cci
```

Reply:
Reflected the review. Please check again.